webapp/src/index.tsx

```diff
@@ -293,21 +294,32 @@ export default class Plugin {
         });
 
         registry.registerWebSocketEventHandler(`custom_${pluginId}_user_reaction`, (ev) => {
+            const reaction = {
+                emoji: {
+                    name: ev.data.emoji_name,
+                    skin: ev.data.emoji_skin,
+                    unified: ev.data.emoji_unified,
+                },
+                timestamp: ev.data.timestamp,
+            };
             store.dispatch({
                 type: VOICE_CHANNEL_USER_REACTION,
                 data: {
                     channelID: ev.broadcast.channel_id,
                     userID: ev.data.userID,
-                    reaction: {
-                        emoji: {
-                            name: ev.data.emoji_name,
-                            skin: ev.data.emoji_skin,
-                            unified: ev.data.emoji_unified,
-                        },
-                        timestamp: ev.data.timestamp,
-                    },
+                    reaction,
                 },
             });
+            setTimeout(() => {
```

Comment:
I'm wondering if we should store this in a queue the same size as the reactions queue, so if there are people reacting like crazy we don't eat up a lot of memory. but that might be me doing premature performance 0/5
Anyway, we can add it later, as my changes are not in the branch yet and I'll have to update the code to match what it is approved already :)

Reply:
Wait. Why don't we simply cancel the previous timeout when a new reaction arrives? We can store the timeout ID in memory along with the reaction and, if there's one already, we just cancel it. We can still check for the timestamp when handling the timeout event to make sure no race condition happens anywhere, but I think that's doable.
Let me try.

Reply:
I'll merge this PR and open a new one with this change.